src/Symfony/Component/Security/Core/Authorization/Voter/AuthenticatedVoter.php

```diff
@@ -28,8 +28,6 @@ class AuthenticatedVoter implements VoterInterface
 {
     public const IS_AUTHENTICATED_FULLY = 'IS_AUTHENTICATED_FULLY';
     public const IS_AUTHENTICATED_REMEMBERED = 'IS_AUTHENTICATED_REMEMBERED';
-    public const IS_AUTHENTICATED_ANONYMOUSLY = 'IS_AUTHENTICATED_ANONYMOUSLY';
-    public const IS_ANONYMOUS = 'IS_ANONYMOUS';
```

Comment:
This attributes should be deprecated (the constants, along with normal string usage)

Reply:
#42510